src/server/api/routers/bot-source.ts

```diff
@@ -402,6 +405,101 @@ async function syncBotSourceSitemap(
     .where(eq(schema.botSources.id, bs.id))
 }
 
+function getSiteMaps() {
+  return publicProcedure
+    .input(
+      z.object({
+        url: z.string(),
+      }),
+    )
+    .query(async ({ input }) => {
+      // add https to url if not present
+      if (!input.url.startsWith('http')) {
+        input.url = `https://${input.url}`
+      }
+
+      // fetch robots.txt of the site
+      let robotsTxt
+
+      try {
+        robotsTxt = await fetch(`${input.url}/robots.txt`)
```

Comment:
So, we are checking if site have site map or not by checking the `{url}.robots.txt`. I think we have 2 problems: 
- How many page have robots.txt. I saw lots of page don't have robots.txt, but they have sitemap.xml or sitemap-*.xml. I think this check is not able to check a page's sitemap yet.
- We can just do this at FE instead.